docs/userguide/constraints.md

```diff
@@ -99,7 +99,7 @@ from baybe.constraints import ThresholdCondition
 
 ThresholdCondition( # will select all values above 150
     threshold = 150, 
-    operator = ">",
+    operator = "=",
```

Comment:
@Scienfitz: I guess this is correct now?

Reply:
took me some time to understand what was even going on here, i think you fixed the wrong thing, the condition here does not fit "will select values above 150" as written in the comment anymore. So imo either i) keep the original simply removing the tolerance or ii) take what you've done here but rewrite the accompanying comment

Reply:
Ah, I see, fixed it. This now resolves #68. I think the fact that this got noticed so quickly by users proves that we need automatic tests 😄